doc/wg/core/notes/core-notes-2022-06-03.md

```diff
@@ -0,0 +1,228 @@
+# Tock Core Notes 2022-06-03
+
+Attendees:
+- Alyssa Haroldsen
+- Brad Campbell
+- Branden Ghena
+- Hudson Ayers
+- Johnathan Van Why
+- Leon Schuermann
+- Pat Pannuto
+- Philip Levis
+- Vadim Sukhomlinov
+
+## Updates
+ * Leon: Small update from my side. I've been working on the DMA buffer story
+   and the process buffer story. Trying to reconcile the two. Writing Miri tests
+   to make sure the solutions are sound in the Rust sense. Implementing the
+   `LeasableBuffer` abstraction we spoke about last week as part of the DMA
+   buffer. Hopefully there will be more news and actual code output next week.
+ * Hudson: Amit is not here but he submitted a PR this past week that addressed
+   some of the concerns with `VolatileCell` that Alyssa had brought up. Alyssa,
+   I hope you're happy with how that ended up?
+ * Alyssa: Yes, it's much better.
+
+## TockWorld
+ * Johnathan: I wanted to check in and make sure we're ready to invite people.
+ * Hudson: Yes, we are ready to send out invites. I'll send out an RSVP form to
+   share. Were you planning to personally invite some of the people you asked to
+   invite?
+ * Johnathan: Yes, that was my plan.
+ * Hudson: Okay, that makes sense.
+ * Johnathan: It sounds like I should wait for the form then I can send invites.
+ * Hudson: I'll paste it into chat.
+ * Phil: It sounds like we are more than just ready to send out invitations --
+   we can send out invitations. [ed: some emphasis was lost when transcribing to
+   text]
+ * Hudson: Yes
+ * Hudson: Johnathan, so we don't duplicate invites, who are you planning to
+   send invites to?
+ * Johnathan: I was planning to invite all the invitees I suggested.
+ * Hudson: Alyssa or Vadim, would you like to send invites to the other Ti50
+   folks?
+ * Alyssa: Yeah, I can. What do I need to get to them, logistically?
+ * Hudson: There's an RSVP form, and you can add some descripton.
+ * Alyssa: Can I send an open invite to the team, so that anyone who wants to
+   come can come?
+ * Hudson: We definitely had a desire to not have too many total people. There
+   was a limit above which Google people would be unable to give talks. I don't
+   want to say no, don't do that, but we didn't really talk about it.
+ * [Some discussion omitted for privacy. Conclusion: Johnathan will invite the
+   people he recommended inviting, Alyssa will invite Ti50 team members]
+ * Hudson: Alyssa, if you want to invite more people from Ti50 that's probably
+   fine but I think Amit should have final say on that and he's not here at the
+   moment.
+ * Alyssa: Okay
+ * Branden: Logistically, we have more space, but I'm concerned that if there
+   are a bunch of random people who are not really interested in Tock it may not
+   be interested in them.
```

Comment:
```suggestion
 * Branden: Logistically, we have more space, but I'm concerned that if there
   are a bunch of random people who are not really interested in Tock it may not
   be interesting to them.
```

I think this is how I remember it being said

Reply:
Yup. Thank you!

Reply:
Yikes, that was a bad transcription error on my part. Yes, that's correct, and I've pushed a fix.